test/nodetool/test_ring.py

```diff
@@ -18,6 +20,12 @@ class Host(NamedTuple):
     state: str
     load: float
     ownership: float
+    tokens: list[str]
```

Comment:
We could share these between `test_ring.py` and `test_status.py` but it is probably not worth it. There won't be a third test needing it.

Reply:
yes, i prefer repeating in this case.